server/index.js

```diff
@@ -48,7 +48,7 @@ app.get('/tiersIncluded/:bundleId', (req, res) => {
 });
 
 app.get('/itemsIncluded/:tierId', (req, res) => {
-  const dataString = req.params.tierId;
+  const dataString = req.params;
   return db.itemsRequest(dataString)
   .then(itemsData => {
     if (itemsData) {
```

Comment:
✓ I see you're sending the params object now instead of just the bundleId got it